lib/sles4sap_publiccloud.pm

```diff
@@ -831,4 +837,105 @@ sub list_cluster_nodes {
     die 'None of the hosts are currently part of existing cluster' unless @cluster_nodes;
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns 1 if hana database is online, 0 otherwise.
+
+=back
+=cut
+
+sub get_hana_database_info {
+    my ($self, $password_db, $instance_id) = @_;
+    my $hdb_cmd = "hdbsql -u SYSTEM -p $password_db -i $instance_id 'SELECT * FROM SYS.M_DATABASES;'";
+
+    sleep 30;
```

Comment:
Usually it's not good to add `sleep` in the code as it may make the test wait always for some seconds when it could wait for an event instead. If the sleep is unavoidable, please add a comment on why `sleep 30` is needed and how you arrived on that number.